src/input/ButtonConfigWidget.cc

```diff
@@ -218,7 +218,7 @@ void ButtonConfigWidget::initComboBox(QComboBox *comboBox, const Settings::Setti
 		comboBox->addItem(icon,desc,actionName);
 	}
 
-	for (std::list<actionStruct>::iterator action=actions.begin(); action != actions.end(); ++action){
+	for (std::list<actionStruct>::const_iterator action=actions.begin(); action != actions.end(); ++action){
```

Comment:
I didn't catch this first time around. Is there a reason this isn't simply `for (const auto &action : actions) {` ?

Reply:
done